lxd/firewall/drivers/drivers_nftables.go

```diff
@@ -265,23 +265,15 @@ func (d Nftables) networkSetupOutboundNAT(networkName string, SNATV4 *SNATOpts,
 }
 
 // networkSetupICMPDHCPDNSAccess sets up basic nftables overrides for ICMP, DHCP and DNS.
-func (d Nftables) networkSetupICMPDHCPDNSAccess(networkName string, ipVersions []uint) error {
-	ipFamilies := []string{}
-	for _, ipVersion := range ipVersions {
-		switch ipVersion {
-		case 4:
-			ipFamilies = append(ipFamilies, "ip")
-		case 6:
-			ipFamilies = append(ipFamilies, "ip6")
-		}
-	}
-
+// This should be called with at least one of (ip4Address, ip6Address) != nil.
+func (d Nftables) networkSetupICMPDHCPDNSAccess(networkName string, ip4Address, ip6Address net.IP) error {
```

Comment:
Please dont compact the types in the functions argument list. This makes things more difficult to read.